.github/workflows/codeql-analysis.yml

```diff
@@ -87,6 +87,23 @@ jobs:
       uses: slackapi/slack-github-action@v1.18.0
       with:
         channel-id: "${{ secrets.SLACK_CHANNEL_ID_FOR_PR_CHECK_NOTIFY }}"
-        slack-message: "@${{ github.actor}} CI Check ${{ job.status }} on ${{ github.event_name }}\n${{ github.event.pull_request.title || github.event.head_commit.message }} \nhttps://github.com/${{github.repository}}/actions/runs/${{github.run_id}}"
```

Comment:
can we have a template or a reusable action for these duplicated sections?

Reply:
I will refactor it later.